ui.apps/src/main/content/jcr_root/apps/etoolbox-backpack/admin/console/page/content/.content.xml

```diff
@@ -109,7 +109,7 @@
                         href="/crx/packmgr"
                         x-cq-linkchecker="valid"
                         text="CRX Package Manager"
-                        variant="actionBar">
+                        icon="Actions">
```

Comment:
Please take into account, that "icon" parameter works with one of the following icons, https://www.adobe.io/experience-manager/reference-materials/6-5/coral-ui/coralui3/Coral.Icon.html#:~:text=CoralUI%20ships%20with%20a%20large%20set%20of%20monochrome%20icons.%20The%20name%20column%20corresponds%20to%20the%20value%20that%20the%20icon%20attribute%20accepts.
See the "Html attribute" column for acceptable values
But do we need an actual icon here, anyway?